media/pages/sk/naco-infozakon/page.html

```diff
@@ -60,6 +60,7 @@ <h2 id="kauza-predrazene-ct-v-piestanskej-nemocnici">Kauza Predražené CT v pie
 zmluvu a ušetriť nemocnici 600 000 eur.
 </p>
 
+<hr>
```

Comment:
Ak spravne chapem navrh vizualu v #68, tak okrem tej ciary ide hlavne o zmensenie pisma a to odsadenie vo footnotach.

Mozno by sa to zislo implementovat novym komponentom/classov. Nieco ako `chv-footenote`. Kde by ta ciara vlastne nebol ani `<hr>` ale top border komponentu.

Pri vypocte presnej velkosti pisma a odsadenie to sprav tak, aby baseline grid bol stale zachovany. A idealne nepouzivaj explicitne konstanty, ale hodnoty odvod z existujucich zakladnych konstant.

Reply:
Text vo footnote v `p` tagu je o 1-2 px vyssie ako grid. Mam tomu spravit este korekciu? Tym ze som puzil `@include chv-baseline` sa zachoval grid system, vsak?

Pre `padding-left` som nenasiel ziadnu vhodnu konstantu.

Reply:
Upravit aj headingy (`<h>` tagy), ktore su vo chv-footnote?

Reply:
> Text vo footnote v `p` tagu je o 1-2 px vyssie ako grid. Mam tomu spravit este korekciu? Tym ze som puzil `@include chv-baseline` sa zachoval grid system, vsak?

Nevyzera to posunute. Text aj vo footnote lezi pekne na grid ciarach:

![footnote](https://user-images.githubusercontent.com/5547275/120530543-ead5c900-c3dd-11eb-8de8-f3197443b57d.png)

Ci si nieco ine myslel? Ano `@include chv-baseline(...)` je presne na to, aby upravil padding/margin odstavca tak, aby text lezal na grid ciarach.

> Pre `padding-left` som nenasiel ziadnu vhodnu konstantu.

To bude asi tym, ze `10px` pre bocny padding je divna konstanta. Skor by som pouzil nieco odvodene od `$chv-list-indent`. Asi `$chv-list-indent / 2`, kedze nepouzivame bullets/ikonky.

Mozno by sa zislo zaviest globalnu konstancu `$chv-indent` a potom `$chv-list-indent` dat ako jej dvojnasobok.

Reply:
Zatial som pouzil `padding-left: ($chv-list-indent / 2)`, nechcel som vytvarat novvu premennu `$chv-indent` pretoze v projekte je zadefinovana classa 
```
.chv-indent
  padding-left: $chv-list-indent
```
Bolo by to potom zle nazvoslovie premennych keby existovala premenna `$chv-indent: 16px` a rovnako volajuca sa classa odkazovala na inu premennu s inou hodnotou. Mozeme to vsak prerobit v novom PR. Aby to bolo znova pouzitelne mozeme spravit mixin, podobne ako `chv-height-{n}`

Reply:
@viliambalaz Jasne. Kludne moze ostat `/ 2` vo footnote. Ak by sme casom mali viac takychto uzkych odsadeni, tak potom tie indent premenne upraceme nejak. Zatial netreba.